nx-X11/programs/Xserver/include/resource.h

```diff
@@ -58,7 +58,6 @@ SOFTWARE.
 typedef unsigned long RESTYPE;
 
 #define RC_VANILLA	((RESTYPE)0)
-#define RC_CACHED	((RESTYPE)1<<31)
```

Comment:
I disagree. resource.h is an internal header within the server. No clients involved.